pgml-extension/src/bindings/transformers.py

```diff
@@ -92,6 +92,11 @@ def embed(transformer, text, kwargs):
     return result
 
 
+def is_embed_cached(transformer):
```

Comment:
I think it'd be better to move this inside the python embed function and raise an exception if it's not cached, so we aren't grabbing the GIL from Rust twice. In the exception case, we can log, call load, and retry.

Reply:
Raise an exception? We don't want to crash `pgml.embed`...

Reply:
Do `print` statements inside transformers.py display while running `pgml.embed`?

Reply:
No, you need to go back to rust to print.